### PR TITLE
Refacto navigation panel strategies

### PR DESCRIPTION
**Pull request by itkg-ngilain**
Add a level 0 strategie + refacto existing files

https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1060
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/33
https://github.com/open-orchestra/open-orchestra-workflow-function-bundle/pull/52

### CODE REVIEW

File: Backoffice/NavigationPanel/Strategies/GSTreeTemplatePanelStrategy.php [hunk 1]
```diff
@@ -18,9 +18,12 @@ class GSTreeTemplatePanelStrategy extends AbstractNavigationPanelStrategy
 
     /**
      * @param TemplateRepositoryInterface $templateRepository
+     * @param string                      $parent
+     * @param                             $weight
```
Comment (itkg-alavieille) [inline Backoffice/NavigationPanel/Strategies/GSTreeTemplatePanelStrategy.php]: the type

File: BackofficeBundle/StrategyManager/NavigationPanelManager.php [hunk 2]
```diff
@@ -35,6 +35,11 @@ public function addStrategy(NavigationPanelInterface $strategy)
      */
     public function show()
     {
+        foreach ($this->strategies as $parent => $weight) {
```
Comment (itkg-alavieille) [inline BackofficeBundle/StrategyManager/NavigationPanelManager.php]: the parameter ``$weight`` isn't very clear, it doesn't only contain the weight

**Comment by itkg-ngilain** [comment]
@itkg-alavieille : updates done

**Comment by itkg-alavieille** [comment]
:+1:

**Pull request merged**